docs/source-2.0/guides/smithy-cli/index.rst

```diff
@@ -0,0 +1,22 @@
+.. _smithy-cli:
+
+==============
+The Smithy CLI
+==============
+
+The Smithy CLI enables you to quickly iterate on your Smithy models. With
+this tool, you can easily build your models, run ad-hoc validation, compare
+models for differences, query models, and more. The CLI makes it incredibly
+easy to get started working with Smithy, without having to worry about setting
+up Java or Gradle.
+
+The :doc:`installation guide <cli_installation>` has all of the information
+you need to get started using the CLI. We will be continuously making
+improvements to the CLI, as well as writing helpful guides to get you started
+utilizing the CLI to its full potential.
```

Comment:
Maybe note how customers can report issues with the CLI? Do we have an issue tag for that?